### DISCUSSION TITLE
Custom Filtering Rules

### DISCUSSION THREAD

**Discussion by idscomm** [AdguardTeam/AdGuardHome#7061] [Q&A]
Hello,

I did some research but could not find the answer to my question therefore I am reaching out to the community. 

I was hoping to find out if there was a way to customize DNS Filtering to only allow (or unblock) entry on a specific website? I tried adding domain=XXXXXX in my rule but it did not appear to work… is this option supported? Is my syntax wrong? -see below:

@@||adobedtm.com^$important,domain=cartersoshkosh.ca

Any help be appreciated.

**Comment by Dynamic5912** [comment]
What exactly is the domain you are trying to unlock/allow?

**Comment by idscomm** [comment]
I was trying to whitelist https://www.cartersoshkosh.ca<https://www.cartersoshkosh.ca/en_CA/home?cm_mmc=sem.can.canada.purchase.carters-ca-text-brand-trademark.71700000112021165...google..carters&gad_source=1&gclid=CjwKCAjwvIWzBhAlEiwAHHWgvS2FUsfuWGwIp70-L1Pu59m8zTG7h1nK-mG3uM9Wi-28vvWIioYAFRoCR5sQAvD_BwE&gclsrc=aw.ds> and all the external sites it’s pulling as my wife shops for the kids and AdGuard is blocking “shopper also like these” purchase suggestion from the site she uses. This page shows an example:
https://www.cartersoshkosh.ca/en_CA/carters-kid-boy-tops/V_3R446714.html

I looked in the logs and quite a few items are blocked. I’m trying to either allow the minimum for the page to load or simply whitelist these links but only for this site (not sure if that’s possible).


Dominic Laflamme
For encrypted communication, my

public PGP Key can be found here:

http://keyserver.pgp.com<http://keyserver.pgp.com/>

PGP Key: 0xCAA9874E

***@***.******@***.***>

Web:http://www.idscomm.ca/

YouTube:https://www.youtube.com/channel/UCeOKGrGmBlAISUBckBjsFjw


VOIP.MS Referal Program<https://voip.ms/en/invite/MjIyNDE3>





-= Sent from my mobile device -=

________________________________

**Reply by Dynamic5912** [reply]
You'll need to whitelist the blocked sites one by one until it starts working then

**Comment by idscomm** [comment]
Ok, il do that then. Thanks for the reply.


Dominic Laflamme
For encrypted communication, my

public PGP Key can be found here:

http://keyserver.pgp.com<http://keyserver.pgp.com/>

PGP Key: 0xCAA9874E

***@***.******@***.***>

Web:http://www.idscomm.ca/

YouTube:https://www.youtube.com/channel/UCeOKGrGmBlAISUBckBjsFjw


VOIP.MS Referal Program<https://voip.ms/en/invite/MjIyNDE3>





-= Sent from my mobile device -=

________________________________
